src/pages/markets/Markets.tsx

```diff
@@ -1,69 +1,68 @@
-import { useMemo, useState } from 'react';
+import { useState } from 'react';
 
-import { Link, useNavigate } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { ButtonAction } from '@/constants/buttons';
 import { STRING_KEYS } from '@/constants/localization';
-import { PREDICTION_MARKET } from '@/constants/markets';
 import { AppRoute, MarketsRoute } from '@/constants/routes';
-import { StatsigFlags } from '@/constants/statsig';
 
 import { useDocumentTitle } from '@/hooks/useDocumentTitle';
 import { usePotentialMarkets } from '@/hooks/usePotentialMarkets';
-import { useAllStatsigGateValues } from '@/hooks/useStatsig';
 import { useStringGetter } from '@/hooks/useStringGetter';
 
 import breakpoints from '@/styles/breakpoints';
 import { layoutMixins } from '@/styles/layoutMixins';
 
 import { Button } from '@/components/Button';
 import { ContentSectionHeader } from '@/components/ContentSectionHeader';
-import { IconName } from '@/components/Icon';
-import { IconButton } from '@/components/IconButton';
 import { Switch } from '@/components/Switch';
 import { MarketsStats } from '@/views/MarketsStats';
 import { MarketsTable } from '@/views/tables/MarketsTable';
 
+import { testFlags } from '@/lib/testFlags';
+
+import { MarketsBanners } from './MarketsBanners';
+
 const Markets = () => {
   const stringGetter = useStringGetter();
   const navigate = useNavigate();
-  const featureFlags = useAllStatsigGateValues();
   const [showHighlights, setShowHighlights] = useState(true);
   const { hasPotentialMarketsData } = usePotentialMarkets();
 
   useDocumentTitle(stringGetter({ key: STRING_KEYS.MARKETS }));
 
-  const marketsPageBanner = useMemo(() => {
-    if (featureFlags?.[StatsigFlags.ffShowPredictionMarketsUi]) {
+  const renderSlotRightActions = () => {
+    if (testFlags.pml) {
+      return (
+        <Button action={ButtonAction.Primary} onClick={() => navigate(AppRoute.LaunchMarket)}>
```

Comment:
feels like these should be links instead of buttons?

Reply:
yeah potentially I can potentially change them to links. It would probably require using the button stylings for them. I think I might also be able to just wrap the Button in `<Link>` or `<a>` tags.